src/rgw/rgw_auth.cc

```diff
@@ -291,6 +307,27 @@ bool rgw::auth::RemoteApplier::is_owner_of(const rgw_user& uid) const
   return info.acct_user == uid;
 }
 
+bool rgw::auth::RemoteApplier::is_identity(
+  const boost::container::flat_set<Principal>& ids) const {
+  for (auto& id : ids) {
+    if (id.is_wildcard()) {
+      return true;
+    } else if (id.is_tenant() &&
+	       (info.acct_user.tenant.empty() ?
```

Comment:
We have some compatibility logic for tenant handling . It's commented in [`rgw_auth.cc`](https://github.com/ceph/ceph/blob/master/src/rgw/rgw_auth.cc#L338-L348).